### PR TITLE
UI - Fix UAVs not being considered vehicles for HUD elements

### LINKED ISSUE #9182: Ammo count digit is lacked for UAV
**Mods (complete and add to the following information):**
- **Arma 3:** `2.12` 
- **CBA:** `3.15.8`
- **ACE3:** `3.15.2`

**Description:**
Ammo count digit is lacked for UAV.

**Steps to reproduce:**
See [screenshot](https://cdn.discordapp.com/attachments/189399686766002176/1094340831655231488/3cad1224b463c6f7.png)

**Expected behavior:**
See [screenshot](https://cdn.discordapp.com/attachments/189399686766002176/1094341798836588725/6e289db95a9ca97e.png)

**Where did the issue occur?**
everywhere

### PR DESCRIPTION
**Pull request by LinkIsGrim**
**When merged this pull request will:**
- Title.
- Close #9182
- Update UI condition for ammo count added in ACE_Reload.
- Change some `count` loops to `forEach`.

### IMPORTANT

- [ ] If the contribution affects [the documentation](https://github.com/acemod/ACE3/tree/master/docs), please include your changes in this pull request so the documentation will appear on the [website](https://ace3.acemod.org/).
- [x] [Development Guidelines](https://ace3.acemod.org/wiki/development/) are read, understood and applied.
- [x] Title of this PR uses our standard template `Component - Add|Fix|Improve|Change|Make|Remove {changes}`.

### CODE REVIEW

**Comment by jonpas** [comment]
> Remove obsolete UI condition added in ACE_Reload. It only forced infantry ammo count to not be shown, disregarding selective UI.

This is supposed to be so. Reload component force hides the ammo count. Ammo count should never be visible with reload component, it makes it redundant.

The other fix is good I think.

**Comment by LinkIsGrim** [comment]
> Ammo count should never be visible with reload component, it makes it redundant.

I'm more of a fan of letting UI handle that exclusively (someone might want reload's features without the forced removal of ammo count), especially since Ammo Count is disabled by default. But that's out of scope for this. Will adjust to revert.

**Comment by jonpas** [comment]
I could agree with default disabled in UI component (I don't remember if it is), if we want to change that.

**Comment by LinkIsGrim** [comment]
It's currently disabled by default in UI, and Reload forces it off regardless of that setting.

File: addons/ui/functions/fnc_setAdvancedElement.sqf [hunk 1]
```diff
@@ -33,9 +33,10 @@ _cachedElement params ["_idd", "_elements", "_location", "_conditions"];
 
 // Exit if main vehicle type condition not fitting
 private _canUseWeaponOrInCargo = ACE_player call CBA_fnc_canUseWeapon || {-1 < vehicle ACE_player getCargoIndex ACE_player};
+private _inUAVCamera = !isNull (ACE_controlledUAV select 0);
```
[the remark this answers is not in the document]
Comment (LinkIsGrim) [reply addons/ui/functions/fnc_setAdvancedElement.sqf:36]: Guess I'll just revert then.

**Pull request merged**